plugins/MultiSites/API.php

```diff
@@ -159,8 +174,14 @@ private function getSitesIdFromPattern($pattern, $_restrictSitesToLogin)
 
         // Both calls above have called Site::setSitesFromArray. We now get these sites:
         $sitesToProblablyAdd = Site::getSites();
+        $idSites = [];
+        if (!empty($sitesToProblablyAdd)) {
+            foreach ($sitesToProblablyAdd as $site) {
+                $idSites[] = $site['idsite'];
+            }
+        }
```

Comment:
Just a minor suggestion that doesn't need to be actioned.

We can omit the check here as `Site` always seem to handle the internal `protected static $infoSites` as an array and that is what `Site::getSites()` return, so should be safe to always iterate over it. If it's empty the foreach loop will do nothing.

```suggestion
        foreach ($sitesToProblablyAdd as $site) {
            $idSites[] = $site['idsite'];
        }
```

Reply:
Had only moved the code around and didn't think too much about it 🙈